packages/x-date-pickers-pro/src/internals/hooks/useEnrichedRangePickerFieldProps.ts

```diff
@@ -83,24 +82,37 @@ export interface RangePickerFieldSlotProps<
   >;
 }
 
+export type RangePickerPropsForFieldSlot<
```

Comment:
All the changes around the range pickers are no longer require, but they improve the readability so I'll extract them in a standalone PR